ipc/cli/src/commands/checkpoint/relayer.rs

```diff
@@ -95,4 +116,16 @@ pub(crate) struct BottomUpRelayerArgs {
         help = "The max parallelism for submitting checkpoints"
     )]
     pub max_parallelism: usize,
+    #[arg(
+        long,
+        default_value = "true",
+        help = "Flag to turn on/off Prometheus metrics reporting."
+    )]
+    pub enable_metrics: bool,
+    #[arg(
+        long,
+        default_value = "0.0.0.0:9185",
+        help = "Local port for exporting Prometheus metrics."
```

Comment:
It's not just a "port". You could either have a host and a port separately, or call it perhaps `metrics_listen_addr` instead.